libs/frontend/feature-render/src/lib/compute-attrs-inputs-mixins.ts

```diff
@@ -54,7 +54,8 @@ export const computeAttrsInputsMixins = (
   // Add attrs and inputs from default `inputs`
   for (const [key, input] of Object.entries(c.inputs ?? {})) {
     if (!inputs[key]) {
-      inputs[key] = resolveInput(input.is ?? input.default)
+      const overrideInput = c.reusableComponentData?.inputs?.[key]
+      inputs[key] = resolveInput(overrideInput ?? input.is ?? input.default)
```

Comment:
If we're letting users override the reusable component input, shouldn't this be:
```ts
inputs[key] = resolveInput(input.is ?? overrideInput ?? input.default)
```